test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -713,6 +712,84 @@ public async Task VerifyCacheFileInsideObjFolder(ProjectTemplate projectTemplate
             }
         }
 
+        [NuGetWpfTheory]
+        [MemberData(nameof(GetPackagesConfigTemplates))]
+        public async Task UpdateAllPCPackagesInPMC(ProjectTemplate projectTemplate)
+        {
+            EnsureVisualStudioHost();
+            using (var simpleTestPathContext = new SimpleTestPathContext())
+            {
+                // Arrange
+                var packageName1 = "UpdateAllPCPackage1";
+                var packageVersion1 = "1.0.0";
+                var packageVersion2 = "2.0.0";
+                await CommonUtility.CreatePackageInSourceAsync(simpleTestPathContext.PackageSource, packageName1, packageVersion1);
```

Comment:
A lot of these calls can probably be inline. 

I see no reason why we can't make a call `

CreatePackagesInSourceAsync and pass a list of package id/version or PackageIdentity. 

It'll significantly reduce the number of lines.

Reply:
We wrote these 2 test case after refering to the existed test case `InstallMultiplePackagesFromPMCForPCAsync()` as below lines. 

```
                var packageName1 = "TestPackage1";
                var packageVersion1 = "1.0.0";
                await CommonUtility.CreatePackageInSourceAsync(testContext.PackageSource, packageName1, packageVersion1);

                var packageName2 = "TestPackage2";
                var packageVersion2 = "1.2.3";
                await CommonUtility.CreatePackageInSourceAsync(testContext.PackageSource, packageName2, packageVersion2);    
```
Given that there are many test cases like that, could we keep the code structure as the original one and update the similar methods (as above) uniformly in future?

Reply:
I think we can start making the changes here. It should be a pretty easy change.